src/RandomProgramGenerator.cpp

```diff
@@ -211,6 +211,9 @@ static void print_help()
 	//--------------------------GCC C Extensions--------------------------
 	cout<< "------------------------------GCC C Extensions------------------------------" << endl << endl;
 	cout << " --function-attributes | --no-func-attributes: enable | disable generate common function attributes (disabled by default)." << endl << endl;
+	cout << " --type-attributes | --no-type-attributes: enable | disable generate common type attributes (disabled by default)." << endl << endl;
+	cout << " --label-attributes | --no-label-attributes: enable | disable generate common label attributes (disabled by default)." << endl << endl;
+	cout << " --variable-attributes | --no-variable-attributes: enable | disable generate common variable attributes (disabled by default)." << endl << endl;
```

Comment:
Can you add an option to enable/disable all attributes? 

Does clang support the attributes here? If the answer is yes, the comment in the front should be `GCC/Clang C Extensions`.

Reply:
I have added an option to enable/disable all attributes as `--compiler-attributes and --no-compiler-attributes`.

No. Clang does not support all of the above attributes because these GCC C Extensions are specific to GCC.

Is PR ready to merge? I have pushed all the suggested changes.